sippy-ng/src/helpers.js

```diff
@@ -30,9 +30,13 @@ export const SafeStringParam = {
 
 // safeEncodeURIComponent wraps the library function, and additionally encodes
 // square brackets.  Square brackets are NOT unsafe per RFC1738, but Google and
-// others mishandle them.
+// others mishandle them.  The encodeURIComponent function will double encode
+// a %20 (space) as %25%20 -- so we back out its double encode.
```

Comment:
I don't think this should be here, safeEncodeURIComponent shouldn't be getting a value that has anything uri encoded in it already.

Reply:
this is now fixed; good suggestion.  I removed the double calls to `safeEncodeURIComponent` and now use it only when making a api call or creating an url